container/docker/templates/conductor-dockerfile.j2

```diff
@@ -10,6 +10,17 @@ RUN yum update -y && \
     yum install -y epel-release && \
     yum install -y gcc git python-devel rsync libffi-devel openssl-devel && \
     yum clean all
+{% elif "rhel" in distro %}
+RUN yum -y update-minimal --disablerepo "*" \
+                          --enablerepo rhel-7-server-rpms,rhel-7-server-optional-rpms \
+                          --security --sec-severity=Important --sec-severity=Critical --setopt=tsflags=nodocs && \
+    yum -y install --disablerepo "*" \
+                   --enablerepo rhel-7-server-rpms \
+		   https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
+    yum -y install --disablerepo "*" \
+                   --enablerepo epel,rhel-7-server-rpms,rhel-7-server-optional-rpms \
+	           gcc git python-devel rsync libffi-devel openssl-devel && \
+    yum clean all
```

Comment:
I keep running into the following error during the conductor build:

```
Rpmdb checksum is invalid: dCDPT(pkg checksums): perl-HTTP-Tiny.noarch 0:0.033-3.el7 - u
```
I couldn't find a way to get yum to ignore the checksum error. The only hack I came up with is to append `; exit 0` to the end of the this whole chained command.